plugin

```diff
@@ -174,7 +175,6 @@
   "marrobHD/tv-card",
   "mathoudebine/homeassistant-browser-control-card",
   "mattieha/select-list-card",
-  "mattieha/slider-button-card",
```

Comment:
Kepp PR to one action (remove/add).

Reply:
Sure, I changed this one for the removal, will create another one for adding the replacement.

Reply:
<https://hacs.xyz/docs/publish/remove#remove-a-repository> Removal instructions

Reply:
Thanks @ludeeus! Third time is hopefully the charm 🤞